packs/classfeatures/flexible-spell-preparation.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "classfeature",
         "description": {
```

Comment:
Typically we only link the first instance of a spell, so the renames shouldn’t be links.

Reply:
Updated, thank you for the guidance

Reply:
The non-link ,emotions should be lower-case to match the source text.

Reply:
Shaved another yak.